rosidl_runtime_rs/src/string.rs

```diff
@@ -3,6 +3,7 @@ use std::ffi::CStr;
 use std::fmt::{self, Debug, Display};
 use std::hash::{Hash, Hasher};
 use std::ops::{Deref, DerefMut};
+use std::ptr;
```

Comment:
I think this can be removed too – it's more consistent with the rest of the code to use the full path below.

Reply:
Done